gluetool/help.py

```diff
@@ -206,12 +227,12 @@ def rst_to_text(text):
     """
     Render given text, written with RST, as plain text.
 
-    :param str text: string to render.
-    :rtype: str
+    :param text text: string to render.
```

Comment:
What is `text` type?

Reply:
It was an attempt to cover all variants of "string" in both Python 2 and 3, but I changed it back to `str` for now.